jbmc/unit/java-testing-utils/require_type.cpp

```diff
@@ -54,40 +54,40 @@ struct_union_typet::componentt require_type::require_component(
 
 /// Checks a type is a code_type (i.e. a function)
 /// \param type: The type to check
-/// \return The cast version of the type code_type
-code_typet require_type::require_code(const typet &type)
+/// \return The cast version of the type method_type
+java_method_typet require_type::require_code(const typet &type)
```

Comment:
:no_entry_sign: I think it would be best to keep this and have another (`require_java_method`), or update the name of the method to be `require_java_method`